pkg/server/server.go

```diff
@@ -389,8 +393,19 @@ func (s *Server) HTTPHandler() http.Handler {
 	handle("/api/console/knative-channels", authHandler(s.handleKnativeChannelCRDs))
 	handle("/api/console/version", authHandler(s.versionHandler))
 
+	// Get running kubernetes version and inject into helm handlers.
+	config := &rest.Config{
+		APIPath:   s.K8sProxyConfig.Endpoint.String(),
+		Transport: s.K8sClient.Transport,
+	}
+	kubeVersion, err := actions.GetKubeVersion(config)
```

Comment:
I guess it's OK to check only once on startup since the API server is upgraded before the other cluster operators. There might be a small window during upgrades where we show charts for the previous version after the API servers are upgraded.

Reply:
Note that I believe the kube version is available as an environment variable in the pod, so we could conceivably avoid needing to make this request (although not for off-cluster mode).

Reply:
@spadgett  Getting kube version from env var `KUBE_GIT_VERSION` in case api server failed to report. Done https://github.com/openshift/console/pull/7012/commits/739ad74f846e52fced4420d57038ce65ef98f449